packages/forms/src/UIForm/fields/Datalist/Datalist.component.test.js

```diff
@@ -154,6 +154,12 @@ describe('Datalist component', () => {
 					...schema.schema,
 					enum: ['my', 'title', 'map'],
 				},
+
+				titleMap: [
+					{ name: 'my_name', value: 'my' },
+					{ name: 'title_name', value: 'title' },
+					{ name: 'map_name', value: 'map' },
+				],
```

Comment:
This doesn't change anything in your test.
The scenario you describe should be tested: 
- no titlemap
- trigger to get the titlemap
- check that on change, titlemap is present